pyvista/plotting/plotting.py

```diff
@@ -5055,12 +5060,15 @@ def add_lines(self, lines, color='w', width=5, label=None, name=None):
         >>> actor = pl.add_lines(points, color='yellow', width=3)
```

Comment:
Tangentially related to this PR, but
1. the yellow-on-white is a very bad choice, I recommend changing this while you're at it
2. the doctest here uses a good example, but the docstring above has `np.array([[0, 0, 0], [1, 0, 0], [1, 0, 0], [1, 1, 0]])` as an example which is ambiguous. I recommend changing this to have 4 distinct points.

Reply:
1. good point - I've changed the line colour to purple
2. I've changed it to use the same coordinates as the doctest